cookbook/tools_examples/requirements.txt

```diff
@@ -0,0 +1 @@
+mirascope>=0.2.0,<1.0.0
```

Comment:
can you do this for other examples as well please?

Reply:
do you want me to raise the min version for examples that havent changed? or only the new ones we added